packages/meditrak-server/src/apiV2/import/importSurveyResponses/importSurveyResponses.js

```diff
@@ -373,9 +494,30 @@ function getColumnHeader(sheet, columnIndex) {
   return getCellContents(sheet, columnIndex, 0);
 }
 
-// Helper to check whether a given column header represents a new survey response to be added
-function checkIsNewSurveyResponse(columnHeader) {
-  return columnHeader && columnHeader.startsWith('NEW');
+function getImportMode(columnHeader) {
+  return Object.values(IMPORT_MODES).find(importMode => columnHeader.startsWith(importMode));
+}
```

Comment:
Although I suppose this can return `undefined` so perhaps we'd need to expand that with an `IMPORT_MODE.DEFAULT` or something similar

Reply:
Good idea, added